README.md

```diff
@@ -8,6 +8,15 @@ CLI tool to interact with Paralus.
 
 `export PATH=$PATH:/usr/local/bin/pctl`
 
+You can download the config using below command (RECOMMENDED)
+
+pctl config download http://console.paralus.local
```

Comment:
Can you move this section to codeblock? Otherwise markdown renderer take this as a single paragraph.

See https://github.com/paralus/cli/tree/v0.1.4_cl#usage

Reply:
good catch @akshay196. moved it to a codeblock now.